utils/helpers.ts

```
@@ -119,25 +119,47 @@ export function getPercentageChangeColor(change: number | 'N/A'): string {
+  const difficultyFactor = Math.round(Number(difficulty) * 100);
+  if (difficultyFactor === 0 || accepted === BigInt(0)) {
+    return {
+      '1h': '<0.001%',
+      '1d': '<0.001%',
+      '1w': '<0.001%',
+      '1m': '<0.001%',
+      '1y': '<0.001%',
+    };
+  }
+
+  const networkDiff = (BigInt(accepted) / BigInt(difficultyFactor)) * BigInt(10000);
```

ESLint

[error] Insert `⏎···`

(prettier/prettier)